publisher_joint_trajectory_controller:
  ros__parameters:

    controller_name: "joint_trajectory_position_controller"
    wait_sec_between_publish: 4

    goal_names: ["pos1", "pos2", "pos3", "pos4"]
    pos1:
      positions: [0.785, 0.785, 0.785]
    pos2:
      positions: [0.0, 0.0, 0.0]
    pos3:
      positions: [-0.785, -0.785, -0.785]
    pos4:
      positions: [0.0, 0.0, 0.0]

    joints:
      - coxa_joint
      - femur_joint
      - tibia_joint

    check_starting_point: false
    starting_point_limits:
      coxa_joint: [-0.1,0.1]
      femur_joint: [-0.1,0.1]
      tibia_joint: [-0.1,0.1]